services/worker/tests/job_runners/split/test_descriptive_statistics.py

```diff
@@ -271,9 +280,9 @@ def descriptive_statistics_string_text_expected(datasets: Mapping[str, Dataset])
     [
         ("descriptive_statistics", None),
         ("descriptive_statistics_string_text", None),
-        ("gated", None),
-        ("audio", "NoSupportedFeaturesError"),
-        ("big", "SplitWithTooBigParquetError"),
+        # ("gated", None),
+        # ("audio", "NoSupportedFeaturesError"),
+        # ("big", "SplitWithTooBigParquetError"),
```

Comment:
why are these commented?

remove them if not needed anymore

Reply:
no they are needed, i comment them to run only the tests i need locally and always forget to uncomment... thanks for noticing!